elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -99,7 +99,10 @@ public void delete(Object object) {
 
     @Override
     public void save(Object object) {
-        deferredTasks.add(() -> session.saveOrUpdate(object));
+        deferredTasks.add(() -> {
+            session.saveOrUpdate(object);
+            session.evict(object);
```

Comment:
Can you provide a comment explaining why we're saving and then evicting the object. I'm assuming this is some magical Hibernate incantation, but it would be nice to provide the explanation so that if it stops working at some point we can have an idea of why this was needed to begin with.